x-pack/plugins/security_solution/common/api/timeline/get_notes/get_notes_route.schema.yaml

```diff
@@ -51,6 +51,11 @@ paths:
           schema:
             type: string
             nullable: true
+        - name: associatedFilter
+          in: query
+          schema:
+            type: string
```

Comment:
Given this is an enum, this should not be a string. Take a look at `TimelineType` in https://github.com/elastic/kibana/blob/7b9ff3d90cabe7e7f9e3ca4e6ecec31344afaff4/x-pack/plugins/security_solution/common/api/timeline/model/components.schema.yaml#L8-L12

Reply:
haaaa nice I was wondering how to do that, thanks I will update!

Reply:
fixed, thanks for pointing this out!